README.md

```diff
@@ -336,3 +336,60 @@ Install a Cilium in a cluster and enable encryption with IPsec
     🚀 Creating Agent DaemonSet...
     🚀 Creating Operator Deployment...
     ⌛ Waiting for Cilium to be installed...
+
+## Experimental `helm` installation mode
+
+`cilium-cli` v0.14 introduces a new `helm` installation mode. In the current installation mode
+(we now call it `classic` mode), `cilium-cli` directly calls Kubernetes APIs to manage resources
+related to Cilium. In the new `helm` mode, `cilium-cli` delegates all the installation state
+management to Helm. This enables you to use `cilium-cli` and `helm` interchangeably to manage your
+Cilium installation, while taking advantage of `cilium-cli`'s advanced features such as Cilium
+configuration auto-detection.
+
+Our current plan is to:
+
+- v0.14: Introduce the `helm` installation mode as an opt-in feature.
+- v0.15: Enable the `helm` installation mode by default, and deprecate the `classic` installation mode.
+- v1.0: Remove the `classic` installation mode.
```

Comment:
The plan SGTM. I wonder whether we should also mention or link this in the *Releases* section above.

Reply:
done ✅ added a link to this section in the `Release` section.